src/frontend/sharedComponents/GPSPill.test.tsx

```diff
@@ -0,0 +1,45 @@
+import {render, screen} from '@testing-library/react-native';
+
+import {GPSPill} from './GPSPill';
+
+test.skip('searching status', async () => {
+  render(<GPSPill status="searching" />);
+
+  // TODO: Check icon color
```

Comment:
checking icon color is a bit tricky, mostly because of how the existing implementation decides to render the icon portion of the pill. first thought is to assign an appropriate accessibility role to the icons used, query for that, and then make an assertion on the styling. not sure if that will work given that one of the icons used is based on the vector icons library.